client/lcd/lcd_test.go

```diff
@@ -177,7 +177,7 @@ func TestBlock(t *testing.T) {
 
 	// --
 
-	res, body = Request(t, port, "GET", "/blocks/1", nil)
+	res, body = Request(t, port, "GET", "/blocks/2", nil)
```

Comment:
Why did this change?

Reply:
OK, we should update gaia lite so that you can easily start it with a separate root-of-trust.

Reply:
Ref https://github.com/cosmos/cosmos-sdk/issues/2323